### DISCUSSION TITLE
JsonContent always instantiates a new MediaTypeHeaderValue when not specified

### DISCUSSION THREAD

**Discussion by paulomorgado** [dotnet/aspnetcore#50959] [Q&A]
When a media type is not specified, [`JsonContent` constructor](https://github.com/dotnet/runtime/blob/2f74c93a0abb69247bce9ad9a40a1604658627dc/src/libraries/System.Net.Http.Json/src/System/Net/Http/Json/JsonContent.cs#L32) always instantiates a new `MediaTypeHeaderValue` through [`JsonHelpers.GetDefaultMediaType()`](https://github.com/dotnet/runtime/blob/ca8d6f0420096831766ec11c7d400e4f7ccc7a34/src/libraries/System.Net.Http.Json/src/System/Net/Http/Json/JsonHelpers.cs#L30).

Is that intended?

Could it be cached?

**Comment by campersau** [comment]
The problem is that it is mutable and others could change it in like a custom `DelegatingHandler` etc.
